timescaledb/how-to-guides/migrate-data/index.md

```diff
@@ -12,6 +12,12 @@ There are a number of choices available to migrate data into TimescaleDB:
 1.  Migrating from InfluxDB: to migrate data from InfluxDB,
     [follow these instructions][outflux]
 
+<highlight type="warning">
+Before you begin migrating data, ensure you have enough free disk space. In most
+cases, you require free space of at least one and a half times the size of the
```

Comment:
I think this falls under the "decimals as numerals" rule in Google's style guide. And if it didn't I would take issue with the guide 😄 I think "1.5" is more readable here

Reply:
```suggestion
cases, you require free space of at least 1.5 times the size of the
```